org.xtext.example.mydsl.ide/src/org/xtext/example/mydsl/ide/EclipseLikeHoverService.xtend

```diff
@@ -0,0 +1,50 @@
+package org.xtext.example.mydsl.ide
+
+import com.google.inject.Inject
+import java.util.List
+import org.eclipse.emf.ecore.EAttribute
+import org.eclipse.emf.ecore.EClass
+import org.eclipse.emf.ecore.EObject
+import org.eclipse.emf.ecore.EStructuralFeature
+import org.eclipse.emf.ecore.util.FeatureMap
+import org.eclipse.xtext.documentation.IEObjectDocumentationProvider
+import org.eclipse.xtext.ide.server.hover.HoverService
+
+class EclipseLikeHoverService extends HoverService {
+	
+	@Inject extension IEObjectDocumentationProvider
+	
+	override def List<? extends String> getContents(EObject element) {
+	    val documentation = element.documentation
+        if (documentation === null) return #[getFirstLine(element)] else #[getFirstLine(element), documentation]
```

Comment:
I like brackets ;) also the `return` if there in the true case, in the else case it is missing